entrypoint.sh

```
@@ -1,35 +1,6 @@
-			rhelmajor=8
-		else
-			log "FATAL ERROR: Unsupported Fedora variant=${VARIANT_ID}"
-			exit 1
-		fi
-		;;
-	*) log "FATAL ERROR: Unsupported OS ID=${ID}"; exit 1
-		;;
-esac
-
-/usr/src/whereabouts/rhel${rhelmajor}/bin/ip-control-loop $@
+/usr/src/whereabouts/bin/ip-control-loop $@
```

Shellcheck (0.11.0)

[error] Double quote array expansions to avoid re-splitting elements.

(SC2068)